media/stylus/components/structure/tabzilla.styl

```diff
@@ -1,34 +1,48 @@
 /*
 Tabzilla Rar!
+V 0.3.0
 ====================================================================== */
 
 #tabzilla {
-    /* below copied from https://github.com/mozilla/bedrock/blob/master/media/css/tabzilla/tabzilla.less#L51-L60 */
-    bidi-value(float, right, left);
-    display: block;
-    height: 44px;
-    width: 150px;
-    overflow: hidden;
-
-    /* Prevents the "mozilla" link from displaying before tabzilla is loaded */
-    opacity: 0;
-
-    &.loaded {
-        opacity: 1;
-    }
+  position: relative;
+  float: right;
 }
-
-#tabzilla-panel {
-    /* in case the javascript loads before the CSS */
-    height: 0;
-    overflow: hidden;
+#tabzilla a {
+  position: relative;
+  display: block;
+  width: 147px;
+  height: 37px;
+  text-indent: 120%;
+  white-space: nowrap;
+  overflow: hidden;
+  background-image: url('/media/img/tabzilla-static.png');
+  background-repeat: no-repeat;
+  z-index: 2;
+}
+@media (-webkit-min-device-pixel-ratio: 1.5), (min--moz-device-pixel-ratio: 1.5), (-o-min-device-pixel-ratio: 3/2), (min-resolution: 1.5dppx) {
+  #tabzilla a {
```

Comment:
Media Queries: resolution feature not supported by: IE (10,11), Safari (8,7.1), iOS Safari (8,8.1-8.4), IE Mobile (10,11)